### PR TITLE
renamed all user list references to uploaded lists

### PR DESCRIPTION
**Pull request by darwishoptly**
@tscanlin @jprovine Have not made a PR in a bit, but I think this is the right process :)

### CODE REVIEW

File: src/content/rest/reference/lists/0-intro.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 template: sidebyside
-title: User lists
+title: Uploaded lists
 anchor: lists
 ---
 
-A user list is a set of user identifiers that you have uploaded to Optimizely. Membership in a targeting list can be used to define an Optimizely audience, so you can target experiments only to a particular set of users.
+A uploaded list is a set of user identifiers that you have uploaded to Optimizely. Membership in a targeting list can be used to define an Optimizely audience, so you can target experiments only to a particular set of users.
```
Comment (tscanlin) [inline src/content/rest/reference/lists/0-intro.md]: Should be "An uploaded list.."

File: src/content/rest/reference/lists/2-create.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ endpoint_prefix: projects/
 endpoint_option: 456
 endpoint_suffix: /targeting_lists
 type: POST
-title: Create a user list
+title: Create a uploaded list
```
Comment (tscanlin) [inline src/content/rest/reference/lists/2-create.md]: Create an uploaded list

File: src/content/rest/reference/lists/2-create.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ response: |
   }
 ---
 
-Create a user list with the given name and comma-separated set of values.
+Create a uploaded list with the given name and comma-separated set of values.
```
Comment (tscanlin) [inline src/content/rest/reference/lists/2-create.md]: an*

File: src/content/rest/reference/lists/3-update.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ endpoint: targeting_lists/123/
 endpoint_prefix: targeting_lists/
 endpoint_option: 123
 type: PUT
-title: Update a user list
+title: Update a uploaded list
```
Comment (tscanlin) [inline src/content/rest/reference/lists/3-update.md]: an*

File: src/content/rest/reference/lists/4-delete.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ endpoint: targeting_lists/123/
 endpoint_prefix: targeting_lists/
 endpoint_option: 123
 type: DELETE
-title: Delete a user list
+title: Delete a uploaded list
```
Comment (tscanlin) [inline src/content/rest/reference/lists/4-delete.md]: an*

**Comment by tscanlin** [comment]
Other than a few cases of changing "a" to "an" this looks good to me. Do you want me to make those changes for you and land this PR once the corresponding change has been made in the app?

**Comment by bhamodi** [comment]
Closing in favour of #116.

**Pull request closed without merging**